x-pack/solutions/observability/plugins/streams_app/public/components/management_bottom_bar/index.tsx

```diff
@@ -60,15 +60,23 @@ export function ManagementBottomBar({
       <EuiButtonEmpty
         data-test-subj="streamsAppManagementBottomBarCancelChangesButton"
         color="text"
-        size="s"
-        iconType="cross"
         onClick={handleCancel}
       >
         {i18n.translate('xpack.streams.streamDetailView.managementTab.bottomBar.cancel', {
-          defaultMessage: 'Cancel changes',
+          defaultMessage: 'Cancel',
         })}
       </EuiButtonEmpty>
-      {confirmButton}
+
+      <EuiButton
+        data-test-subj="streamsAppManagementBottomBarButton"
+        disabled={disabled}
+        color="primary"
+        fill
+        onClick={onConfirm}
+        isLoading={isLoading}
+      >
+        {confirmButton}
+      </EuiButton>
```

Comment:
Here you should leave it as it was and update the button on [line 37](https://github.com/elastic/kibana/pull/211704/files#diff-72b3664be0ebafd46a6f30f91fc66f8df183628c0e16de1b38ce82d9141f45d3R37)